packages/admin/admin/src/common/FieldSet.tsx

```diff
@@ -0,0 +1,149 @@
+import ArrowForwardIosSharpIcon from "@mui/icons-material/ArrowForwardIosSharp";
+import { ComponentsOverrides, Theme } from "@mui/material";
+import MuiAccordion, { AccordionProps } from "@mui/material/Accordion";
+import MuiAccordionDetails, { AccordionDetailsProps } from "@mui/material/AccordionDetails";
+import MuiAccordionSummary, { AccordionSummaryProps } from "@mui/material/AccordionSummary";
+import { createStyles, WithStyles, withStyles } from "@mui/styles";
+import clsx from "clsx";
+import * as React from "react";
+
+interface FieldSetComponentsProps {
+    accordion?: Partial<AccordionProps>;
```

Comment:
MUI recommends to name the outermost components/slot `root`, so I'd suggest we name it `root` here as well: https://mui.com/material-ui/guides/creating-themed-components/#1-create-the-component-slots

> Though you can give these slots any names you prefer, we recommend using root for the outermost container element for consistency with the rest of the library.